ipn/ipnstate/ipnstate.go

```diff
@@ -38,6 +38,11 @@ type Status struct {
 	TailscaleIPs []netaddr.IP // Tailscale IP(s) assigned to this node
 	Self         *PeerStatus
 
+	// Health contains health check problems.
+	// Empty means everything is good. (or at least that no known
+	// problems are detected)
+	Health []string
```

Comment:
Not parsable by frontends in this form.

If someday we want to report health to particular network nodes, and render it nicely in the UI next to the node, we'll want something more here. Should we pre-empt it and make a `type Health struct { Problem []string }` to add fields to?

Reply:
for per-peer health, we can put that on the Peer struct itself, no?

Reply:
that is, on `Peer map[key.Public]*PeerStatus`

Reply:
sounds good